frontend/components/pod_components/HomeScreen.js

```diff
@@ -2,46 +2,76 @@ import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { StyleSheet, Text, View, SafeAreaView, TouchableOpacity } from 'react-native';
 import Constants from 'expo-constants';
+import axios from 'axios';
 
-export default function HomeScreen({ navigation }) {
-    return (
+import ProgressBar from '../ProgressBar.js';
+
+const server_add = Constants.manifest.extra.apiUrl;
+
+export default class HomeScreen extends React.Component{
+    state = {
+        Trainee_progress: 0, 
+        Associate_progress: 0,
+        Partner_progress: 0
+    };
+
+    componentDidMount(){
+        axios.get(server_add + 'calculateProgressBar', {
+            params: { // Currently using fake data 
+                firstname : 'Fake',
+                lastname : 'F',
+                email: 'fakef@gmail.com'
+            }
+        })
+        .then(response => {
+            console.log(response.data);
+            let data = response.data;
+            this.setState({
+                // Currently only have fake data on Trainee
+                Trainee_progress: data.records[0].TR_CareerExpl_Outcomes__c + data.records[0].TR_Competency_Outcomes__c + data.records[0].TR_LifeEssentials_Outcomes__c, 
+                Associate_progress: 0,
+                Partner_progress: 0
+            })
+        })
+        .catch(function (error) {
+            console.log(error);
+        });
+    }
+
+    render(){
+        return(
         <SafeAreaView style={styles.container}>
-            <Text style={styles.title} >
-                HOME SCREEN
+        <TouchableOpacity 
+            style={styles.block} 
+            onPress={() => this.props.navigation.navigate('Trainee Pod')}
+        >
+            <Text style={styles.blockText}>
+                Trainee 
+            </Text>
+            <ProgressBar progress={this.state.Trainee_progress} total_tasks={7} />
```

Comment:
It would be better if the total_tasks was not hard coded into the params of ProgressBar. It would be better if you were able to call an endpoint that could return the number of tasks associated with the pod, incases someone the MTW team adds or removes task. This applies to lines 56 and 66 as well.